README.md

```diff
@@ -319,25 +318,27 @@ curl -v --header "Content-Type: application/json" \
   http://<service-aca-uri>/ContosoChat
 ```
 
-## Costs
-You can estimate the cost of this project's architecture with [Azure's pricing calculator](https://azure.microsoft.com/pricing/calculator/)
+## Guidance
+
+### Region Availability
+
+This template uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
```

Comment:
@cassiebreviu please update this generic section with actual models and region information before merge.